source/_UIAHandler.py

```diff
@@ -320,19 +323,26 @@ def getNearestWindowHandle(self,UIAElement):
 			# Called previously. Use cached result.
 			return UIAElement._nearestWindowHandle
 		try:
-			window=UIAElement.cachedNativeWindowHandle
+			processID=UIAElement.cachedProcessID
+		except COMError:
+			return None
+		appModule=appModuleHandler.getAppModuleFromProcessID(processID)
+		# WDAG (Windows Defender application Guard) UIA elements should be treated as being from a remote machine, and therefore their window handles are completely invalid on this machine.
+		# Therefore, jump all the way up to the root of the WDAG process and use that window handle as it is local to this machine.
+		if appModule.appName==u'hvsirdpclient':

```

Comment:
I think it might be handy to pull the process name out and store it in a class or module level constant. The name might be handy when someone searches for `WDAG_PROCESS_NAME` or similar